docs/docs/guides/remote-access.md

```diff
@@ -57,3 +61,4 @@ A remote reverse proxy like [Cloudflare](https://www.cloudflare.com/learning/cdn
 
 - Complex configuration
 - Depending on your configuration, both the Immich web interface and API may be exposed to the internet. Immich is under very active development and the existence of severe security vulnerabilities cannot be ruled out.
+- Reverse proxy [can weaken the privacy](https://www.youtube.com/results?search_query=cloudflare+privacy) of your information.
```

Comment:
Yes, I agree with you, but maybe because this is such an easy option to implement through cloudflare, it might be worth informing users that this is not necessarily a way that guarantees privacy, since cloudflare can view information as unencrypted if they want to.

Reply:
I get that, but I think I would prefer putting the focus on Cloudflare (and similar products) then and don't generalize it this much